modules/installation-about-custom-azure-vnet.adoc

```diff
@@ -79,6 +79,11 @@ The network security group rules must be in place before you install the cluster
 |x
 |x
 
+|`22623`
+|Allows communication to the machine config server.
+|x
+|x
+
```

Comment:
Hello,

My understanding is that the control nodes and bootstrap are serving the ignition config files on port 22623, but not on the compute nodes.

Do you have an example or reference for the port 22623 requirement on the compute nodes?

References: https://docs.openshift.com/container-platform/4.5/installing/installing_azure/installing-azure-account.html#installation-azure-limits_installing-azure-account
https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-cluster-machines_installing-aws-user-infra

Reply:
HI @e-tienne,

You're right. Only bootstrap and masters serves the `ignition` files on that port (in fact, there's a LoadBalancer in front of them).
But anyway, worker nodes needs to access this port also in order to get the proper ignition file to be able to start the OS.
In the same way, this table shows port **6443** for compute nodes, and as you may know, compute nodes don't publish anything in that port (API service is a control plane service), but of course, they need access to the API port.

Reply:
@e-tienne, with the additional context from @josgonza-rh, are you ok with me applying this change to 4.4+?

Reply:
Hello Kathryn,

I think we agree about what needs to be explained, but I think we should expand on the traffic direction.

In this document, I don't see the direction of the traffic which is not intuitive as a user (you need to know both the source and destination for proper granularity on firewall rules).

The control nodes are a destination for port 6443 and 22623, and the source is the control nodes as well as the compute nodes. I would personally drop the 2 `X` under Compute, unless we can add SRC and DST to that table so we can explain the direction of the traffic as I would assume that we're talking about Destination ports.

Example: **Port**: 22623, **Destination**: Control Plane, **Source**: Control Plane & Compute

Reply:
Hi @e-tienne,

I'm completely agree with you about to be more accurate regarding the ports needs (dest and source). In fact I think with should have the same info that we had in OpenShift 3.11 [Required Ports](https://docs.openshift.com/container-platform/3.11/install/prerequisites.html#required-ports).

But IMHO, both suggestions are out of the scope of the aim of this PR and I think this deserves to open an issue to face that. But In the midtime, I'd merge this PR that it is continuist with the current format of the table.

Reply:
Hi @josgonza-rh,

I'd be ok with the changes, conditional on opening a DOCS RFE similar to the one you just linked for 3.11.

Reply:
That would be great. 
@e-tienne @kalexand-rh, please do the needful. Let me know if there's anything else you need from my side.